### DISCUSSION TITLE
Reporting Valve opening

### DISCUSSION THREAD

**Discussion by jarcikw** [SukramJ/aiohomematic#952] [Q&A] [answered]
Does integration report valve opening value on Hmip-etrv-2 ?

1.30.2 - Integration version
2023.2.0 - home assistant
3.67.10  - CCU3 version
1.22 - XML-API

**Comment by stuckis6er** [comment, marked as answer]
Yes, but the entity is disabled. The entity level can be activated in the device settings.

**Reply by jarcikw** [reply]
thanks !
